.github/workflows/builds.yml

```diff
@@ -67,17 +67,29 @@ jobs:
               run_tests: false
             build: Release
           - platform:
-              name: ALPAKA
+              name: "ALPAKA CPU"
               container: ghcr.io/acts-project/ubuntu2404:69
               options: --preset alpaka-fp32
               run_tests: true
             build: Release
           - platform:
-              name: ALPAKA
-              container: ghcr.io/acts-project/ubuntu2404:69
-              options: --preset alpaka-fp32
+              name: "ALPAKA CUDA"
+              container: ghcr.io/acts-project/ubuntu2404_cuda:69
+              options: --preset alpaka-fp32 -Dalpaka_ACC_GPU_CUDA_ENABLE=ON
               run_tests: false
-            build: Debug
+            build: Release
+          - platform:
+              name: "ALPAKA HIP"
+              container: ghcr.io/acts-project/ubuntu2404_rocm_oneapi:69
+              options: --preset alpaka-fp32 -Dalpaka_ACC_GPU_HIP_ENABLE=ON
+              run_tests: false
+            build: Release
+          - platform:
+              name: "ALPAKA SYCL"
```

Comment:
Unfortunately the quotes get lost when `"ALPAKA SYCL"` is given as the argument to `ci_setup.sh`. :frowning:

`SYCL` has to appear in the name for that script to do the right thing. :thinking: I would propose using either `"SYCL ALPAKA"` (so that `SYCL` would appear first), or `"ALPAKA_SYCL"` (so that it would be a single word).

I agree, neither of these are super nice. :frowning: At least we should make the names consistent. If `"ALPAKA_SYCL"` works for you, you should add underscores in the names of the other platforms as well as part of this PR.